pkg/cli/mapping_server.go

```diff
@@ -120,7 +120,14 @@ func (c *MappingServerCommand) RunUnstarted(ctx context.Context, args []string)
 	}
 	closer = multicloser.Append(closer, assetClient.Close)
 
-	processor, err := processors.NewAssetInventoryProcessor(ctx, assetClient, fmt.Sprintf("projects/%s", c.cfg.ProjectID))
+	var defaultScope string
+	if c.cfg.ResourceScope == "" {
```

Comment:
Should we change the naming to DefaultResourceScope?

Reply:
In the current implementation, the DefaultResourceScope is the project scope. If we decided to make the `ResourceScope` a required value, I think we can rename this to `DefaultResourceScope`, but if we intend to keep the current behavior, I don't think the renaming is necessary.

Reply:
This resource scope is only used when we cannot find the resource scope in the resource name, so it is more like a default, naming it ResourceScope is confusing?